packages/ketcher-core/src/application/editor/actions/polymerBond.ts

```diff
@@ -0,0 +1,46 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+import { ReStruct } from 'application/render';
+import { Action } from 'application/editor';
+import { PolymerBondAdd } from 'application/editor/operations/polymerBond';
+import { PolymerBondDelete } from 'application/editor/operations/polymerBond/polymerBondDelete';
+import { PolymerBond } from 'domain/entities/PolymerBond';
+
+type Data = {
+  firstMonomer;
+  startPosition;
+  endPosition;
+};
+
+export function fromPolymerBondAddition(
+  renderersContainer: ReStruct,
+  data: Data,
+): Action {
+  const action = new Action();
+  action.addOp(new PolymerBondAdd(data)).perform(renderersContainer);
```

Comment:
Could you please rename this function to 'addOperation'?

Reply:
This method is used in 140+ places in code. Maybe it should be done in separate PR